### DISCUSSION TITLE
Some questions about android and ios apps

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#1750] [General]
Feodor, I have some questions about android and ios apps that were not clear to me:
1 - All application logic can be done in python with flet, in this case can I use PYPI libraries?
2 - Is there a script for accessing native resources such as camera, notifications, gps, sensors?
3 - Is my python code safe in the APK?

I ask this because I wouldn't want to use an API for my application, like web development with Flet.

Congratulations on the job!!

**Comment by plugeit** [comment]
@FeodorFitsner please reply
